src/Migrate/test/Set-AzMigrateHCIServerReplication.Tests.ps1

```diff
@@ -0,0 +1,22 @@
+if(($null -eq $TestName) -or ($TestName -contains 'Set-AzMigrateHCIServerReplication'))
+{
+  $loadEnvPath = Join-Path $PSScriptRoot 'loadEnv.ps1'
+  if (-Not (Test-Path -Path $loadEnvPath)) {
+      $loadEnvPath = Join-Path $PSScriptRoot '..\loadEnv.ps1'
+  }
+  . ($loadEnvPath)
+  $TestRecordingFile = Join-Path $PSScriptRoot 'Set-AzMigrateHCIServerReplication.Recording.json'
+  $currentPath = $PSScriptRoot
+  while(-not $mockingPath) {
+      $mockingPath = Get-ChildItem -Path $currentPath -Recurse -Include 'HttpPipelineMocking.ps1' -File
+      $currentPath = Split-Path -Path $currentPath -Parent
+  }
+  . ($mockingPath | Select-Object -First 1).FullName
+}
+
+Describe 'Set-AzMigrateHCIServerReplication' {
+    It 'ByID' -skip {
+        $output = Set-AzMigrateHCIServerReplication -TargetObjectID $env.hciProtectedItem2 -TargetVMName $env.hciTgtVMNameEdit -SubscriptionId $env.hciSubscriptionId
+        $output.Count | Should -BeGreaterOrEqual 1
+    }
+}
```

Comment:
```suggestion
Describe 'Set-AzMigrateHCIServerReplication' -Tag 'LiveOnly' {
    It 'ByID' {
        $output = Set-AzMigrateHCIServerReplication -TargetObjectID $env.hciProtectedItem2 -TargetVMName $env.hciTgtVMNameEdit -SubscriptionId $env.hciSubscriptionId
        $output.Count | Should -BeGreaterOrEqual 1
    }
}
```

Reply:
Please add LiveOnly tag for this test case as it needs to install Az.Resources. 

Describing Set-AzMigrateHCIServerReplication
    [-] ByID 633ms
      RuntimeException: Az.Resources Module must be installed to run this command. Please run 'Install-Module -Name Az.Resources' to install and continue.
      at CheckResourcesModuleDependency<Process>, /mnt/vss/_work/1/s/artifacts/Debug/Az.Migrate/custom/Helper/CommonHelper.ps1: line 22
      at Set-AzMigrateHCIServerReplication<Process>, /mnt/vss/_work/1/s/artifacts/Debug/Az.Migrate/custom/Set-AzMigrateHCIServerReplication.ps1: line 137
      at Set-AzMigrateHCIServerReplication<Process>, /mnt/vss/_work/1/s/artifacts/Debug/Az.Migrate/exports/ProxyCmdletDefinitions.ps1: line 6689
      at <ScriptBlock>, /mnt/vss/_work/1/s/artifacts/Debug/Az.Migrate/test/Set-AzMigrateHCIServerReplication.Tests.ps1: line 19